amplify-tools-gradle-plugin/src/main/groovy/com/amplifyframework/tools/gradle/plugin/AmplifyTools.groovy

```diff
@@ -0,0 +1,219 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class AmplifyTools implements Plugin<Project> {
+    void apply(Project project) {
+        def doesNodeExist = true
+        def doesGradleConfigExist
+
+        def profile = 'default'
+        def accessKeyId = null
+        def secretAccessKey = null
+        def region = null
+        def envName = 'amplify'
+        def syncEnabled = 'true'
+
+        project.task('verifyNode') {
+            try {
+                project.exec {
+                    commandLine 'which', 'node'
+                    standardOutput = new ByteArrayOutputStream()
+                }
+            } catch (e) {
+                doesNodeExist = false
+                println("Node is not installed. Visit https://nodejs.org/en/download/ to install it")
+            }
+        }
+
+        project.task('createAmplifyApp') {
+            doesGradleConfigExist = project.file('amplify-gradle-config.json').isFile()
+            if (doesNodeExist && !doesGradleConfigExist) {
+                project.exec {
+                    commandLine 'npx', 'amplify-app', '--platform', 'android'
+                }
+            }
+        }
+
+        project.task('getConfig') {
+            def inputConfigFile = project.file('amplify-gradle-config.json')
+            if (inputConfigFile.isFile()) {
+                def configText = inputConfigFile.text
+                def jsonSlurper = new groovy.json.JsonSlurper()
```

Comment:
Can you import this, so you can just use `JsonSlurper()` without the full qualification?